docs/Tutorials/Merge-Testnet.md

```diff
@@ -160,4 +207,5 @@ After starting Besu and Teku, your node should start syncing and connecting to p
         2022-03-21 20:44:12.353 INFO  - Syncing     *** Target slot: 76096, Head slot: 3519, Remaining slots: 72577, Connected peers: 9
         ```
 
+You can check your validator status by searching your Ethereum address on the [Prater Beacon Chain explorer](https://prater.beaconcha.in/).
```

Comment:
Can we add a note that explains that **Prater** and **Goerli** and two names for the same chain?

Reply:
Goerli's beacon chain is now up and running, so I just removed Prater completely